main.sh

```diff
@@ -667,8 +667,8 @@ for tool in "${tools[@]}"; do
     fi
     # cargo-udeps 0.1.30 and wasm-pack 0.12.0 do not support --version option.
     case "${tool}" in
-        biome) rx "${tool_bin}" --version || true ;; # biome up to 1.2.2 exits with 1 on --version and  --help
-        cargo-careful | cargo-machete) ;;            # cargo-careful 0.3.4 and cargo-machete 0.5.0 do not support neither --version nor --help option.
+        biome) rx "${tool_bin}" --version || true ;;                 # biome up to 1.2.2 exits with 1 on --version and  --help
```

Comment:
I think the space added before comment could be the reason why tidy has failed

Reply:
Actually I'm pretty sure tidy has failed due to this line:
```
Error: file-permissions-check failed: executable should be in tools/ directory
```

I added the spaces when tidy told me to :)

Reply:
> ```
> Error: file-permissions-check failed: executable should be in tools/ directory
> ```

This is due to the fact that you have added a version to codegen. See https://github.com/taiki-e/install-action/issues/302.